machine/v1beta1/0000_10_awsplacementgroup.crd.yaml

```diff
@@ -0,0 +1,176 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
does this make sense to include in SNO?

Reply:
They are talking about adding multiple workers to a single-node master.  Let's keep the API surface consistent.